gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -140,6 +148,32 @@ private int runInternal(FileSystem fs, String cmd, Map<String, String> cmdArgs)
     throw new IllegalArgumentException("Unknown command: " + cmd);
   }
 
+  private int benchmarkWrite(FileSystem fs, Map<String, String> args) {
+    if (args.size() < 1) {
+      System.err.println(
+          "Usage: write"
+              + " --file=gs://${BUCKET}/path/to/test/dir/"
+              + " [--total-size=<file size to write in bytes>]"
+              + " [--write-size=<write buffer size in bytes>]"
+              + " [--num-writes=<number of times to fully write test file>]"
+              + " [--num-threads=<number of threads to run test>]");
+      return 1;
+    }
+
+    Path testFile = new Path(args.get("--dir"));
+
+    benchmarkFileIO(
+        fs,
+        testFile,
+        false,
```

Comment:
please add comments on argument names

Reply:
replaced boolean with enum. This is more descriptive